src/axom/quest/InOutOctree.hpp

```diff
@@ -2634,10 +2634,10 @@ class InOutOctreeMeshDumper
 
 
     // Add the fields to the mint mesh
-    VertexIndex* vertID = addIntField(debugMesh, "vertID" );
-    VertexIndex* lLevel = addIntField(debugMesh, "level" );
+    auto* vertID = addIntField(debugMesh, "vertID" );
```

Comment:
I think it would be more clear to use `axom::IndexType*` instead of `auto*`

Reply:
I used `auto` since:
* We don't particularly care about the type as long as it is correct
* In this case in particular (`InOutOctree`), I wasn't sure if I wanted to keep them tied to ``axom::IndexType`` or to make them user settable.  I.e. the user would be able to set a parameter to control if the octree is using 64 bit or 32 bit indexing, regardless of the configuration of `axom::IndexType`.  (see my PR summary comment)
   * So, using `auto` seemed like it would introduce the least amount of change.

I agree that it is a bit inconsistent since I had to use the explicit types for the arrays, so I'll update the code to be more consistent.

Reply:
Thanks @kennyweiss -- I think not using `auto` in this case will make the code cleaner and more consistent.